aws/aws.go

```diff
@@ -34,6 +34,9 @@ const (
 	awsDevicePrefixWithH = "/dev/hd"
 	awsDevicePrefixNvme  = "/dev/nvme"
 	contextTimeout       = 30 * time.Second
+
+	// DryRunOption is the key to tell if dry run the request
+	DryRunOption = "--dry-run"
```

Comment:
Instead of taking an AWS specific flag, let's define this at the cloudops interface level and a constant like `DryRun`. 
We should look at that flag in the aws provider and convert it to whatever aws wants. And ignore for the rest of the providers.